circle.yml

```diff
@@ -586,6 +586,13 @@ jobs:
           command: |
             rustfmt --version
             cargo fmt --all -- --check
+      - install_wabt
+      - run:
+          name: Check wat2wasm4tests
+          command: |
+            rustfmt --version
+            ./wat2wasm4tests.py bindings/rust/src/lib.rs
```

Comment:
Why not use `find` here too (like for C++)?

Reply:
That would be good to.